### ISSUE TITLE
[Suggestion] Make indicator light flash if not connected to anything

### ISSUE THREAD

**Issue by Sullos** [tsunamayo/Starship-EVO#2282]
### Short Description of the suggestion:

Upon using the indicator light a bit around ships now, I have come to think it would be good if there was an indication if the indicator light was not connected to anything. This would be good so you could keep track of systems that have been destroyed in combat for example, or if you've been sabotaged.
So if the indicator light isn't linked to anything it could perhaps flash red, or not have any color at all. The same goes for if the indicator light is no longer attached to something functional, so an indicator light attached to a reactor. Then if the reactor is destroyed, the indicator light could flash red or just turn off, to allow people to see if a system is offline on their ship with ease.

Also as a reminder, a flat version of the indicator light like a button would be very handy.

Someone also suggested that having an F menu on the indicator light would be good. So we can set the color of the light, and its behaviors, like flashing, on or off, etc.

**Comment by tsunamayo** [comment]
Yes all of that makes a lot of sense, might be a use for my Input Gate (can trigger logic on a specific event, here the destruction of the block). Thanks
